app/models/passenger.rb

```diff
@@ -29,6 +29,13 @@ class Passenger < ApplicationRecord
     if active_status_changed? && archived?
       PassengerMailer.notify_archived(self).deliver_now
     end
+    if active_status_changed? && active?
+      PassengerMailer.send_brochure(self).deliver_now
+    end
+  end
+
+  after_create do
+    PassengerMailer.notify_pending(self).deliver_now
```

Comment:
When a passenger is created by a dispatcher (which is the only way it can happen right now), their status is not pending. We should only be sending the "your account is pending" email upon creation _if_ their account is pending.

Reply:
I should modify what I said above to say that their account _can_ be active if created by a dispatcher--of course if the dispatcher selects "pending" as their status, that's not the case. But much of the time it is.

So if they get created and their account is active, they'll get two emails, which isn't ideal.